modules/core/src/rand.cpp

```diff
@@ -734,6 +734,12 @@ cv::RNG& cv::theRNG()
     return getCoreTlsData().get()->rng;
 }
 
+void cv::setRNGSeed(int seed)
+{
+    getCoreTlsData().get()->rng.state = static_cast<uint64>(seed);
```

Comment:
I believe, you shouldn't expose implementation details: neither TLS storage, nor RNG internals like "state".
Something like this:
```
theRNG() = RNG(seed);
```